index.d.ts

```diff
@@ -442,8 +442,14 @@ declare namespace Moleculer {
 		error?: (ctx: Context<any, any>, err: Error) => Promise<void> | void;
 	}
 
+	interface RestSchema{
+		path: string,
+		method: 'GET' | 'POST' | 'DELETE' | 'PUT' | 'PATCH'
```

Comment:
Please cover all available options. You can find it in tests: 
https://github.com/moleculerjs/moleculer-web/blob/master/test/services/test.service.js#L38
https://github.com/moleculerjs/moleculer-web/blob/master/test/services/test.service.js#L49
https://github.com/moleculerjs/moleculer-web/blob/master/test/services/test.service.js#L290